admin_manual/maintenance/upgrade.rst

```diff
@@ -2,93 +2,48 @@
-notification it brings you here, to this page.
-
-**It is best to keep your ownCloud server upgraded regularly**, and to install 
-all point releases and major releases without skipping any of them, as skipping 
-releases increases the risk of errors. Major releases are 8.0, 8.1, 8.2, and 
-9.0. Point releases are intermediate releases for each major release. For 
-example, 8.0.9 and 8.1.3 are point releases. **Skipping major releases is not 
-supported.**
-
-**Upgrading is disruptive**. Your ownCloud server will be put into maintenance 
-mode, so your users will be locked out until the upgrade is completed. Large 
-installations may take several hours to complete the upgrade.
-
-.. warning:: **Downgrading is not supported** and risks corrupting your data! If 
-   you want to revert to an older ownCloud version, make a new, fresh 
-   installation and then restore your data from backup. Before doing this, 
-   file a support ticket (if you have paid support) or ask for help in the 
-   ownCloud forums to see if your issue can be resolved without downgrading.
-
-.. not sure about notifications
-.. Update Notifier and Updater App Are Not the Same
-.. ------------------------------------------------
-
-.. ownCloud has two update tools: the ownCloud core update notifier, and the 
-.. Updater app. Figure 1 shows what you see when the Updater app is enabled: 
-.. both 
-.. the core notifier and the Updater app control panel are visible on your 
-.. admin 
-.. page.
-
-.. .. figure:: images/2-updates.png
-..   :alt: Both update mechanisms displayed on Admin page.
-   
-..   *Figure 1: The top yellow banner is the update notifier, and the Updates 
-..   section is the Updater app.*
-   
-.. The core update notifier has only one function, and that is to display a 
-.. notification when a new ownCloud release is available. Then you decide which 
-.. upgrade method to use. When you maintain your ownCloud server via your Linux 
-.. package manager you should ensure that the Updater app is disabled.
+Before beginning an upgrade, please keep the following points in mind:
+
+- Review :doc:`the release notes </release_notes>` for important information about the needed migration steps during that upgrade to help ensure a smooth upgrade process.
+- Skipping major releases is not supported. However *you can* migrate from 8.2 or 9.0 straight to 10.0.   
+- Downgrading is not supported.
+- Upgrading is disruptive, as your ownCloud server will be put into :ref:`maintenance mode <maintenance_mode_label>`. 
+- Large installations may take several hours to complete the upgrade.
+- Downgrading **is not supported** as it risks corrupting your data. If you want to revert to an older ownCloud version, make a new, fresh installation and then restore your data from backup. Before doing this, file a support ticket (if you have paid support) or ask for help in the ownCloud forums to resolve your issue without downgrading.
```

Comment:
fwiw, I added a supplementary point in 6e83d658 addressing this point, which GH doesn't seem to have picked up.